src/ebmc/transition_system.h

```diff
@@ -18,9 +19,19 @@ class message_handlert;
 class transition_systemt
 {
 public:
+  transition_systemt()
+    : trans_expr{
+        ID_trans,
+        conjunction({}),
+        conjunction({}),
+        conjunction({}),
+        typet()}
+  {
+  }
```

Comment:
1.  Is there some place where this should now be used?
2. I am on the fence whether the empty conjunction is helpful here or not: it makes clear that these operands should eventually hold conjunctions, but does hide that this is just a fancy way of producing `true_exprt`.

Reply:
Replaced by `true_exprt()`